src/include/binder/binder.h

```diff
@@ -286,6 +298,10 @@ class Binder {
     function::TableFunction* getScanFunction(
         common::FileType fileType, const common::ReaderConfig& config);
 
+    void extractExportData(
+        std::string selQuery, std::string tableName, std::vector<ExportedTableData>& exportData);
+    void getExportInfo(const catalog::Catalog& catalog, std::vector<ExportedTableData>& exportData);
```

Comment:
No need to create a function unless you are reusing it or the logic should clearly be isolated from parent function.

As a principal, we wanna keep header file simpler and try to hide as many functions as possible into cpp files. So I wouldn't create this `getExportInfo`